exercises/collatz-conjecture/example.exs

```diff
@@ -0,0 +1,25 @@
+defmodule CollatzConjecture do
+
+  @doc """
+  calc/1 takes number (> 1), and returns the number of steps required to get to 1 when
+  following the rules: if n is odd, multiply with 3 and add 1. if n is even, divide by 2
+  """
+  @spec calc(number :: integer) :: pos_integer
+  def calc(number) when number < 1 do
+    raise ArgumentError, "Only positive numbers are allowed"
+  end
+
+  def calc(number) do
```

Comment:
Since the previous scenario returns an error if invalid data is supplied, that would (imo) be the best place to check for an integer